docs/getting_started/modin_vs_dask.rst

```diff
@@ -0,0 +1,104 @@
+be implemented, but it would be inefficient, and column-wise operations such as ``apply(axis=0)`` 
+would be impossible to implement in Dask's architecture.
+
+Delayed Computation
+___________________
+Dask DataFrame makes use of lazy evaluation, which means that the computation is delayed until users explicitly evaluate the results. This mode of evaluation places a lot of optimization responsibility on the user. Specifically, Dask DataFrame's API differs from pandas in that it requires users to explicitly call ``.compute()`` to materialize the result of the DataFrame computation. The need to explicitly trigger computation makes the API less convenient to work with, but allows Dask to perform holistic optimizations over the entire dataflow graph.
+
+Non-order preserving
+_____________________
+By default, pandas preserves the order of the DataFrame, so that users can expect a
+consistent, ordered view as they are operating on their DataFrame. However, Dask makes no
+guarantees about the order of rows in the DataFrame. This is because Dask sorts the 
+``index`` for optimization purposes to speed up computations that involve the row index, as 
+a result, it can not support user-specified order.
+
+Modin
+"""""
+
+Comprehensive API support
+_________________________
+Modin is targeted toward parallelizing the entire pandas API and currently implements `more than 90% <https://github.com/modin-project/modin#pandas-api-coverage>`_ of the pandas API. Modin acts as a drop-in replacement for pandas, such that even if the API is not yet
+supported, it still works by falling back to running pandas. One of the key features
+of being a drop-in replacement is that not only will it work for existing code, if a
+user wishes to go back to running pandas directly, they are not locked in to using Modin and can switch between Modin and pandas at no cost. In other words, Modin notebooks can be converted to and from pandas as the user desires.
+
+.. figure:: ../img/api_coverage_comparison.svg
+   :align: center
+   :alt: Percentage coverage of the pandas dataframe API after deduplication
+
+Eager evaluation
+________________
+Eager evaluation is the default mode of operation for data scientists when working with pandas DataFrames in an interactive environment, such as Jupyter Notebooks. Modin reproduces this familiar behavior by performing all computations as soon as it is issued, so that users can inspect intermediate results and quickly see the results of their computations without having to wait or explicitly trigger computation. This is especially useful during interactive data analysis, where users often iterate on their DataFrame workflows or build up their DataFrame queries in an incremental fashion. Modin also supports lazy evaluation via the OmniSci engine, you can learn more about it :doc:`here </flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index>`.
```

Comment:
```suggestion
________________

Eager evaluation is the default mode of operation for data scientists when working with pandas DataFrames in an interactive environment, such as Jupyter Notebooks. Modin reproduces this familiar behavior by performing all computations as soon as it is issued, so that users can inspect intermediate results and quickly see the results of their computations without having to wait or explicitly trigger computation. This is especially useful during interactive data analysis, where users often iterate on their DataFrame workflows or build up their DataFrame queries in an incremental fashion. Modin also supports lazy evaluation via the OmniSci engine, you can learn more about it :doc:`here </flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index>`.
```